ooipy/hydrophone/basic.py

```diff
@@ -196,7 +201,7 @@ def compute_spectrogram(self, win="hann", L=4096, avg_time=None, overlap=0.5, ve
                     nperseg=L,
                     noverlap=int(L * overlap),
                     nfft=L,
-                    average="median",
+                    average=average_type,
```

Comment:
The same change should be be made in line 224

Reply:
Okay, I think I've made this change. Do you have some easy to check, broadband files to see if the compute_spectrogram_mp, works for both mean and median?